modules/uhttp/client/client.go

```diff
@@ -39,16 +40,18 @@ var (
 // Client wraps around a http client
 type Client struct {
 	*http.Client
+	info                auth.CreateAuthInfo
```

Comment:
Yes, (sadly) otherwise http client will not be accessible outside for use @madhuravi